modules/testserver/src/main/java/com/smartbear/readyapi4j/testserver/execution/TestServerTestStepResult.java

```diff
@@ -22,11 +22,18 @@
     @Override
     public HarEntry getHarEntry() {
         if (harEntry == null && !hasCheckedForHarEntry) {
-            HarLogRoot logRoot = execution.getTestServerApi().getTransactionLog(execution.getId(),
-                    testStepResultReport.getTransactionId(), execution.getAuth());
-
-            if (hasHarEntry(logRoot)) {
-                harEntry = logRoot.getLog().getEntries().get(0);
+            HarLogRoot logRoot = null;
+            try {
+                logRoot = execution.getTestServerApi().getTransactionLog(execution.getId(),
+                        testStepResultReport.getTransactionId(), execution.getAuth());
+
+                if (hasHarEntry(logRoot)) {
+                    harEntry = logRoot.getLog().getEntries().get(0);
```

Comment:
yes - there will only be one entry when requesting the transaction log for a specific transactionId - @prakashmrec correct?